scalyr_agent/tests/scalyr_monitor_test.py

```diff
@@ -74,7 +74,6 @@ def test_int_conversion(self):
         self.assertRaises(BadMonitorConfiguration, self.get, 2.0, convert_to=int)
         self.assertRaises(BadMonitorConfiguration, self.get, True, convert_to=int)
         self.assertRaises(BadMonitorConfiguration, self.get, "12a", convert_to=int)
-        self.assertRaises(BadMonitorConfiguration, self.get, 4, convert_to=int)
```

Comment:
After automated conversion(it was 4L), this operation will not raise an error. Removed to pass the test.